### ISSUE TITLE
Feedback for the microprofile-fallback guide

### ISSUE THREAD

**Issue by tt-le** [OpenLiberty/cloud-hosted-guide-converter#56]
# Step 2

- [x] The following should be reworded and we should also prompt the user to open another terminal before prompting them to `curl`.
<img width="320" alt="Screen Shot 2021-01-11 at 6 37 15 PM" src="https://user-images.githubusercontent.com/44335153/104250656-085cc300-543c-11eb-9184-b507f0826ef9.png">

- [x] `start` should be changed to `finish`

<img width="317" alt="Screen Shot 2021-01-11 at 6 40 53 PM" src="https://user-images.githubusercontent.com/44335153/104250868-8c16af80-543c-11eb-894f-056bb6e4cf37.png">

- [x] parsing issue
<img width="317" alt="Screen Shot 2021-01-11 at 6 42 44 PM" src="https://user-images.githubusercontent.com/44335153/104251006-cc762d80-543c-11eb-8949-284db5e20f65.png">

- [x] Same thing here; `start` should be changed to `finish`, the parsing issue also happens after this `curl` command

<img width="317" alt="Screen Shot 2021-01-11 at 6 46 40 PM" src="https://user-images.githubusercontent.com/44335153/104251260-5aeaaf00-543d-11eb-9b62-6e9def125d07.png">

# Step 5

- [x] Should reword the following: 

<img width="401" alt="Screen Shot 2021-01-11 at 7 07 14 PM" src="https://user-images.githubusercontent.com/44335153/104252498-38a66080-5440-11eb-9067-2fd1e71cb52b.png">

- [x] Add a `curl -k -u admin https://localhost:9443/metrics/application` command which prompts users to enter the `adminpwd` password
<img width="402" alt="Screen Shot 2021-01-11 at 7 11 38 PM" src="https://user-images.githubusercontent.com/44335153/104252768-d732c180-5440-11eb-9293-a51e365663ad.png">

- [x]  This issue happens a couple times in this step:
<img width="397" alt="Screen Shot 2021-01-11 at 7 12 16 PM" src="https://user-images.githubusercontent.com/44335153/104252795-edd91880-5440-11eb-97de-ace19a01e10f.png">

- [x] Wording here should be changed because we are using `curl` as opposed to using the browser

<img width="429" alt="Screen Shot 2021-01-11 at 7 13 51 PM" src="https://user-images.githubusercontent.com/44335153/104252878-2678f200-5441-11eb-9f93-b7e26761bb2a.png">

- [x] Wording here should be changed as well.
- [ ] The -I flag should be added to the command so that the status code shows up ie. `curl -I http://localhost:9080/system/properties`
```
HTTP/1.1 503 Service Unavailable
X-Powered-By: Servlet/4.0
Date: Tue, 12 Jan 2021 00:28:55 GMT
Content-Length: 0
Content-Language: en-US
Connection: Close
```
<img width="429" alt="Screen Shot 2021-01-11 at 7 29 17 PM" src="https://user-images.githubusercontent.com/44335153/104253772-4e695500-5443-11eb-9ef4-d598c988b228.png">

- [x] Missing a `curl -k -u admin https://localhost:9443/metrics/application` 

# Step 6

- [x] The integration test code has comments that are not well formed which causes errors

The application works, integration test passed when I copied the code from the draft site and log in also works!

<img width="498" alt="Screen Shot 2021-01-11 at 7 42 19 PM" src="https://user-images.githubusercontent.com/44335153/104254574-1f53e300-5445-11eb-83f5-0c72552a8c14.png">

**Comment by jakub-pomykala** [comment]
I have created a new issue in regards to creating a function for allowing the guideConverter to create different `curl` commands depending on some conditions.

**Comment by tt-le** [comment]
## Step 2
Still unchanged: `start` should be `finish`

<img width="518" alt="Screen Shot 2021-01-20 at 2 46 41 PM" src="https://user-images.githubusercontent.com/44335153/105226673-50f83880-5b2e-11eb-93c0-58d6f4b29b74.png">

## Step 5 
Still has some curl commands in the middle of the paragraph, will this be fixed or would this require hard coding?

## Step 6 
Still has the same issue with the code block

**Comment by jakub-pomykala** [comment]
Step 2 should be start, this is because if you read the instruction the user is working from the start dir, not the finish therefore he is going to update what is in the start with the code given bellow

**Comment by jakub-pomykala** [comment]
I have now fixed Step 2, closing this issue

**Comment by tt-le** [comment]
For step 5, the application doesn't really work unless we use that new curl command I wrote `curl -k -u admin https://localhost:9443/metrics/application` because the guide requires "authentication" at that url

**Comment by jakub-pomykala** [comment]
Best solution would be to implement `ifdef` into the guide @gkwan-ibm?

**Comment by gkwan-ibm** [comment]
yes, someone need to create the PR on the static guide, and mark this issue pending on it. 
When all issues are addressed, then we can close this issue.

**Comment by jakub-pomykala** [comment]
Step 5 now has a `curl -k -u admin https://localhost:9443/metrics/application` command.

**Comment by jakub-pomykala** [comment]
@tt-le in regards to 
> The -I flag should be added to the command so that the status code shows up ie. curl -I http://localhost:9080/system/properties

Is it all the `http://localhost:9080/system/properties` need them or just one of them?

**Comment by tt-le** [comment]
@jakub-pomykala Just that one command needs the `-I` flag. I just double checked and it's the only one asking you to look at the status code

**Comment by jakub-pomykala** [comment]
https://github.com/OpenLiberty/guide-microprofile-fallback/pull/161

**Comment by jakub-pomykala** [comment]
Since the above PR has been merged this issue can now be closed.
